src/functions/chat/Help/index.js

```diff
@@ -33,7 +33,7 @@ module.exports = async function(context, req) {
   // }
 
   const message =
-    "I can respond to the following commands: !attention, !blog, !discord, !fart, !font, !github, !heroines, !hype, !keyboard, !pobox, !project, !sfx, !so {user name}, !team, !theme, !twitter, !website, !youtube";
+    "I can respond to the following commands: !attention, !blog, !discord, !fart, !font, !github, !giving, !heroines, !hype, !keyboard, !pobox, !project, !sfx, !so {user name}, !team, !theme, !twitter, !website, !youtube, !vulcan";
```

Comment:
V before Y 😄

Reply:
He doesn't know words yet he knows the alphabet - hecc 😜